gui/src/components/modalServer.vue

```diff
@@ -612,6 +625,7 @@ export default {
       v: "",
       allowInsecure: false,
       protocol: "vmess",
+      key: "none",
```

Comment:
1. Does none as the default value of key work?
1. I noticed that key should be encoded, could you please take a testing for 'key'? Thank you.

Reply:
I referred to the v2ray-core documentation: https://www.v2ray.com/en/configuration/transport/quic.html#quicobject

1. The value `none` for `key` should work, as long as `key` is also not specified on the server side. The documentation mentioned above states that `none` is the default value for `key`.
2. I'm not sure that `key` needs to be encoded. I tested the connection using `key` on my server, and everything works fine. It only fails if the `key` value is incorrect.

Thank you!